site/content/integrate/slash-commands/built-in/_index.md

```diff
@@ -35,7 +35,8 @@ Each Mattermost installation comes with some built-in slash commands that are re
 | /shortcuts                                            | Display a list of keyboard shortcuts                                                              | /shortcuts                                                 |
 | /shrug *{message}*                                    | Add `¯\_(ツ)_/¯` to your message                                                                   | /shrug oh well                                             |
 | /status *{emoji_name}* *{descriptive status_message}* | Set a custom status that includes an optional emoji and a descriptive status message              | /status sick Feeling unwell and taking time off to recover |
-| /status clear                                         | Clear the current status                                                                          | /status clear                                              |
+| /status clear                                         | Clear the current status                                                                          | /status clear
+| /marketplace                                          | Open the Mattermost Marketplace model                                                             | /marketplace
```

Comment:
I'm not sure what "Mattermost Marketplace model" is referring to. Does this mean the in-product Marketplace?

Reply:
It does indeed! Great catch! 
@justinegeffen In the Developer documentation, we've been distinguishing between the 2 marketplaces by "in-product" and "web". Any concerns with this approach from your perspective?

Reply:
Ah, I didn't see this before I reviewed, sorry! I'm not actually sure I know the difference between the two.

Reply:
This slash command specifically opens the in-product Marketplace, and doesn't take users to the web-based Marketplace via https://mattermost.com/marketplace/. With this in mind, I'm going to leave the term "product" in the docs.